generators/generator_json

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import json
+import glob
+
+try:
+    tests_dir = os.environ['TESTS_DIR']
+    conf_dir = os.environ['CONF_DIR']
+except KeyError:
+    print("Export the TESTS_DIR and CONF_DIR variables first")
+    sys.exit(1)
+
+name = ''
+fname = ''
+templ = ''
+vals = []
+
+for cfg in glob.glob(os.path.join(conf_dir, 'generators', '*.json')):
```

Comment:
maybe we could use a subdir for all the inputs for this generator? I think it is possible that other generators may want to use jsons for configuration too.

Reply:
I agree, a subdir should be used to make this more future proof